adapters/repos/db/vector/hnsw/distancer/l2_amd64_test.go

```diff
@@ -83,17 +124,23 @@ func Benchmark_L2_PureGo_VS_AVX(b *testing.B) {
 				y[i] = r.Float32()
 			}
 
-			b.Run("pure go", func(b *testing.B) {
-				for i := 0; i < b.N; i++ {
-					L2PureGo(x, y)
-				}
-			})
-
-			b.Run("asm AVX", func(b *testing.B) {
-				for i := 0; i < b.N; i++ {
-					asm.L2(x, y)
-				}
-			})
+			for i := 0; i < b.N; i++ {
+				// L2PureGo(x, y)
+				// asm.L2(x, y)
+				asm.L2AVX512(x, y)
+			}
+
+			// b.Run("pure go", func(b *testing.B) {
+			// 	for i := 0; i < b.N; i++ {
+			// 		L2PureGo(x, y)
+			// 	}
+			// })
+
+			// b.Run("asm AVX", func(b *testing.B) {
+			// 	for i := 0; i < b.N; i++ {
+			// 		asm.L2(x, y)
+			// 	}
+			// })
```

Comment:
is this commented part needed?

Reply:
It's commented to simplify generating benchstat results, but it's not really needed. I'll remove that